src/data/customer/Route.js

```diff
@@ -322,3 +322,32 @@ export const getRouteAsync = async (routeId, token) => {
     return apiResponseErrorHandler(e);
   }
 };
+
+/**
+ * Ger Route Location Order
+ * @param {int} routeId
+ * @param {int} routeLocationId
+ * @param {string} token
+ * @return {Object} Promise resolve/reject
+ //TODO: needs unit testing
```

Comment:
Shouldnt we be adding that first @pncsantos ?

Reply:
@maartenhemmes , added unit test.